exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function first
+function divide(a,b) {
+    return a/b
```

Comment:
As a thought, what would happen if `b = 0`? Your code would try and execute `a / 0` - try this and see what happens. This leads onto a nice way of programming called "defensive programming"